src/main/resources/static/common/js/demoAdmin.js

```diff
@@ -837,28 +856,28 @@ mod.controller('demo_admin_controller', function($rootScope, $scope, $http, $fil
 					}else{
 						messageNotFound = true;
 					}
-				}else if(status === 400){
-					if(message.indexOf("in stock") !== -1){
-						demo.itemModal.formError = "IN_STOCK_NUMBER_IS_INCORRECT";
-					}else if(message.indexOf("exists") !== -1){
-						demo.itemModal.formError = "ITEM_NAME_ALREADY_EXISTS";
-					}else if(message.indexOf("item name") !== -1 && message.indexOf("empty") !== -1){
-						demo.itemModal.formError = "ITEM_NAME_CAN_NOT_BE_EMPTY";
-					}else if(message.indexOf("description") !== -1){
-						demo.itemModal.formError = "DESCRIPTION_VALUE_IS_INCORRECT";
-					}else if(message.indexOf("image file") !== -1 && message.indexOf("empty") !== -1){
-						demo.itemModal.formError = "IMAGE_PATH_CAN_NOT_BE_EMPTY";
-					}else if(message.indexOf("category") !== -1 && message.indexOf("null") !== -1){
-						demo.itemModal.formError = "CATEGORY_CAN_NOT_BE_EMPTY";
-					}else if(message.indexOf("image") !== -1 && message.indexOf("supported") !== -1){
-						demo.itemModal.formError = "UNSUPPORTED_FORMAT";
-					}else{
-						messageNotFound = true;
-					}
-				}else if(status === 500){
-					if(message.indexOf("regiontype") !== -1){
+				}else if(status === 400 || (isGraphql && itemsDTO.region !== "" && (itemsDTO.inStock === "" || itemsDTO.inStock <= (Math.pow(2, 31) - 1)))){
```

Comment:
Why do we need to create itemsDTO for condition checks? 
I can see for graphql case it also include `status` field. 
Maybe instead of checking `message`, check `data.message` for graphql case.

![image](https://user-images.githubusercontent.com/35082985/202808248-6b054f12-948f-4967-8c77-fefaf09942bd.png)